src/configure/mod.rs

```diff
@@ -145,19 +146,24 @@ pub fn JobConfigureImap(context: &Context) {
             // Step 2: Autoconfig
             4 => {
                 progress!(context, 200);
-                if param.mail_server.is_empty()
+
+                if let Some(new_param) = get_offline_autoconfig(context, &param) {
```

Comment:
i think, we should use the provider-database only if the user has not entered a port, server eg. manually. otherwise, if the provider-database contains errors, there is no way for the user to override the settings.